README.md

```diff
@@ -69,17 +65,36 @@ units. One unit represents a payout of one token
 contract is resolved.
 
 The position table uses negative numbers for UNFIXED
-positions, positive numbers for FIXED.  There are no
+positions, positive numbers for FIXED. There are no
 zero positions.
 
-
 ## Getting started
 
+### On Windows
+
+Windows 10. Docker version 20.10.12, build e91ed57. Visual Studio Code v1.65.
+
+Ensure to have downloaded Docker and follow the runs on their platform.
+
+On the installation of docker, run `docker build .`.
```

Comment:
Can you explain why this is needed as a separate step?  There is already a `docker build` command line in each of the 2 shell scripts below.